src/progenitor/progenitordata.cpp

```diff
@@ -83,6 +84,18 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   auto S_adm_dev = S_adm.getOnDevice();
   auto Srr_adm_dev = Srr_adm.getOnDevice();
 
+  // Post-processing params
+  Real outside_pns_threshold =
+      pin->GetOrAddReal("progenitor", "outside_pns_threshold",
+                        2.42e-5); // corresponds to entropy > 3 kb/baryon
+  Real inside_pns_threshold = pin->GetOrAddReal("progenitor", "inside_pns_threshold",
+                                                0.008); // corresponds to r < 80 km
+  UnitConversions units(pin);
+  Real LengthCGSToCode = units.GetLengthCGSToCode();
+  Real rc = 400e5 * LengthCGSToCode;
+  auto mdot_radii = pin->GetOrAddVector("progenitor", "mdot_radii",
+                                        std::vector<Real>{rc}); // default 400km
```

Comment:
I don't think you want to conversion here.
```suggestion
  Real LengthCGSToCode = units.GetLengthCGSToCode();
  auto mdot_radii = pin->GetOrAddVector("progenitor", "mdot_radii",
                                        std::vector<Real>{400}); // default 400km
```